src/it/scala/com/exasol/cloudetl/orc/OrcDataImporterIT.scala

```diff
@@ -0,0 +1,264 @@
+package com.exasol.cloudetl.orc
+
+import java.nio.charset.StandardCharsets.UTF_8
+import java.sql.Timestamp
+
+import com.exasol.cloudetl.BaseDataImporter
+import com.exasol.matcher.CellMatcherFactory
+import com.exasol.matcher.ResultSetStructureMatcher.table
+import com.exasol.matcher.TypeMatchMode._
+
+import org.apache.hadoop.hive.ql.exec.vector._
+import org.apache.orc.OrcFile
+import org.apache.orc.TypeDescription
+import org.apache.hadoop.conf.Configuration
+
+class OrcDataImporterIT extends BaseDataImporter {
+
+  override val schemaName = "ORC_SCHEMA"
+  override val bucketName = "orc-bucket"
+  override val dataFormat = "orc"
+  var conf = new Configuration
+
+  test("imports boolean") {
+    OrcChecker("struct<f:boolean>", "BOOLEAN", "boolean_table")
+      .withInputValues(List(true, false, null))
+      .assertResultSet(
+        table()
+          .row(java.lang.Boolean.TRUE)
+          .row(java.lang.Boolean.FALSE)
+          .row(null)
+          .matches()
+      )
+  }
+
+  test("imports byte") {
+    OrcChecker("struct<f:tinyint>", "DECIMAL(9,0)", "byte_table")
+      .withInputValues(List(11, null))
+      .assertResultSet(
+        table().row(java.lang.Byte.valueOf("11")).row(null).matches(NO_JAVA_TYPE_CHECK)
+      )
+  }
+
+  test("imports short") {
+    OrcChecker("struct<f:smallint>", "DECIMAL(9,0)", "short_table")
```

Comment:
Is it okay that both byte and short have DECIMAL(9,0)?

Reply:
Yes, this fine. Decimal column shows the destination column type in the Exasol table, smallint and shortint both map to decimal(9,0).

Reply:
But tinyint can be decimal(3,0) from the docs, changed it.